### PR TITLE
added GPS by Ayutac

### PR DESCRIPTION
**Pull request by Ayutac**
crafted with basic display over a compass. use to show coordinates.

Tested, works.

### CODE REVIEW

File: src/main/java/techreborn/items/GpsItem.java [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+				.append(Text.literal(" Z:").formatted(Formatting.GRAY))
+				.append(Text.literal(String.valueOf(pos.getZ())).formatted(Formatting.GOLD)));
+			return new TypedActionResult<>(ActionResult.SUCCESS, stack);
+		}
+		return new TypedActionResult<>(ActionResult.PASS, stack);
+	}
+
+	@Override
+	public void appendTooltip(ItemStack stack, @Nullable World worldIn, List<Text> tooltip, TooltipContext flagIn) {
+		getPos(stack)
```
Comment (modmuss50) [inline src/main/java/techreborn/items/GpsItem.java]: Question: Where is this position wirtten to the ItemStack?

Comment (Ayutac) [reply src/main/java/techreborn/items/GpsItem.java]: oops! I couldn't figure it out and intended to delete that. This has been copied from the transmitter, so it is wrong there as well I suppose. On it!

**Review by modmuss50 — APPROVED** [review]
Nothing wrong with this, just not too sure on how useful it may be. Quite happy to merge this but I wonder if it could be a little more intresting 🤔

**Comment by Ayutac** [comment]
you are free to extend on it ;)

It is semi-useful as of now because of F3, but F3 is a debug tool and coordinates are important enough that they should be available ingame without F3 imo. And since GPS is tech and this is TR, I thought it would be a neat little addition :)

**Pull request merged**